app/services/inline_uploads.rb

```diff
@@ -197,19 +197,25 @@ def self.match_anchor(markdown, external_href: false)
     end
   end
 
-  def self.match_img(markdown, external_src: false)
+  def self.match_img(markdown, external_src: false, uploads: nil)
     markdown.scan(/(<(?!img)[^<>]+\/?>)?(\s*)(<img [^>\n]+>)/i) do |match|
-      node = Nokogiri::HTML5::fragment(match[2].strip).children[0]
+      # rubocop:disable Discourse/NoNokogiriHtmlFragment
+      node = Nokogiri::HTML::fragment(match[2].strip).children[0]
```

Comment:
Why was this change required?

Reply:
I just tried this one again and it looks like it was a problem with the example I used for testing. In my development environment, that email was parsed differently by `Nokogiri::HTML` and `Nokogiri::HTML5`. The change is no longer needed. :blush: